packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenuItemWrapper/ContextualMenuButton.test.tsx

```diff
@@ -46,6 +46,50 @@ describe('ContextualMenuButton', () => {
       expect(onClickMock).toHaveBeenCalledTimes(1);
       expect(onClickMock).toHaveBeenCalledWith(menuItem, expect.objectContaining(mockEvent));
     });
+
+    it('does not update when props values do not change', () => {
+      let reactCalls: number = 0;
+      jest.spyOn(ContextualMenuButton.prototype, 'render').mockImplementation(() => {
+        reactCalls += 1;
+        return null;
+      });
```

Comment:
@ddlbrena would you mind changing these mocks you've added to be `jest` mocks which can be asserted against using `toHaveBeenCalledTimes`, see https://jestjs.io/docs/en/expect#tohavebeencalledtimesnumber.

Will remove the need for an explicit counter. We use them in `List` tests for example: https://github.com/OfficeDev/office-ui-fabric-react/blob/2948f3d6aba40e03e5fafd5de06459597eccd9af/packages/office-ui-fabric-react/src/components/List/List.test.tsx#L125

Reply:
I'll make the change this evening if you're unable to get to it.

Reply:
I can do it, I will send an update in a bit.

Reply:
Done.

Reply:
Awesome! I pushed up a slight revision that simplifies the test code, see https://github.com/OfficeDev/office-ui-fabric-react/pull/8360/commits/842f3dfb3dc2643e43c50693cd052159739cf5f9.